nginx/1.25.1/alpine/3.18.2/tpl/Makefile

```diff
@@ -19,19 +23,22 @@ core:
 # ##############################################################################
 # we're on an architecture upstream doesn't officially build for
 # let's build binaries from the published packaging sources
-# TODO: need to get all patches from openresty?
-# TODO: Find a better way to fallback when openresty is behind the nginx latest version and no patch is available
 	curl -sLo /nginx.tar.gz https://nginx.org/download/nginx-${VER_NGINX}.tar.gz
 	tar -C / -xvzf /nginx.tar.gz
 	mkdir -p /var/cache/nginx/client_temp \
 		/var/cache/nginx/proxy_temp \
 		/var/cache/nginx/fastcgi_temp \
 		/var/cache/nginx/uwsgi_temp \
-		/var/cache/nginx/scgi_temp
+		/var/cache/nginx/scgi_temp \
+		/nginx-${VER_NGINX}/patches
+	git clone https://github.com/openresty/openresty.git
+	PATCH_VER=${VER_NGINX}; \
+	if [ "$(shell ls -1 openresty/patches/nginx-${VER_NGINX}-*.patch)" = "" ]; then \
```

Comment:
Potential issue with expanding the wildcard, might be a useful workaround iterate with a loop and.

	PATCH_VER=${VER_NGINX}
	if [ "$(ls -1 openresty/patches/nginx-${VER_NGINX}-*.patch 2>/dev/null)" = "" ]; then
	    PATCH_VER=${FALLBACK_VER_NGINX}
	fi
	for file in openresty/patches/nginx-${PATCH_VER}-*.patch
	do
	    cp "$file" /nginx-${VER_NGINX}/patches/
	done